Tests/PostgreSQLTests/PostgreSQLConnectionTests.swift

```diff
@@ -451,18 +459,39 @@ class PostgreSQLConnectionTests: XCTestCase {
 
 extension PostgreSQLConnection {
     /// Creates a test event loop and psql client.
-    static func makeTest() throws -> PostgreSQLConnection {
-        let hostname: String
+    static func makeTest(transportConfig: PostgreSQLTransportConfig? = nil) throws -> PostgreSQLConnection {
         #if Xcode
-        hostname = (try? Process.execute("docker-machine", "ip")) ?? "192.168.99.100"
+        let hostname = (try? Process.execute("docker-machine", "ip")) ?? "192.168.99.100"
+        let hostnameSSL = hostname
+        let portSSL = 5433
+        let password: String? = nil
         #else
-        hostname = "localhost"
+        let hostname = "localhost"
+        let hostnameSSL = "localhost-ssl"
```

Comment:
Really awesome solution with this 👍. I will try to emulate this with `docker-compose` in the future so that we don't need the `#if Xcode` here.